### PR TITLE
Add supervision minister and save filter to urls

### PR DESCRIPTION
**Pull request by nicolaskempf57**
During the development, the cache sent me the same response without treating the query parameter.
I fixed it.

Closes https://github.com/etalab/data.gouv.fr/issues/1109
Closes https://github.com/etalab/data.gouv.fr/issues/1108

### CODE REVIEW

File: frontend/site/.vitepress/theme/composables/dataProxy.js [hunk 1]
```diff
@@ -96,10 +96,6 @@ export default function useDataProxy(mapping, statuses) {
         });
     }
 
-    onMounted(() => {
-        getData();
```
Comment (maudetes) [inline frontend/site/.vitepress/theme/composables/dataProxy.js:100]: I may be missing something, but seems like `getData` is never called anymore? I had to add the onMounted back to make it work locally

File: frontend/site/.vitepress/theme/composables/useUpdateUrl.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    return {
+        DONT_SAVE_TO_HISTORY,
+        NEXT_CURSOR,
+        SAVE_TO_HISTORY,
+        getSearchParams,
+        updateUrl,
+    };
+}
+
+export function getSearchParams() {
```
Comment (maudetes) [inline frontend/site/.vitepress/theme/composables/useUpdateUrl.js:42]: When coming from a filtered URL, nextCursor has some value, leading to a Load More to show up. Ex: http://localhost:5173/?supervisor=Minist%C3%A8re+du+Travail+du+Plein+emploi+et+de+l%27Insertion

**Review by maudetes — COMMENTED** [review]
Nice!

File: frontend/site/.vitepress/theme/composables/dataProxy.js (hunk 1 above)
Comment (nicolaskempf57) [reply frontend/site/.vitepress/theme/composables/dataProxy.js:100]: I moved it to InventoryBase but my condition is wrong without a pasted link : https://github.com/etalab/ouverture.data.gouv.fr/pull/31/files#diff-730b9322507594a4427a84fb7710a8b229714ef9e57534deb52cc4db8066b0f4R198

**Pull request merged**